lib/modules/feed_press/feed_press.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * This module handles the FeedPress auth methods and provides access to 
+ * all Feeds analyzed by FeedPress. To fetch the data two static methods 
+ * can be used:
+ *
+ * fetch_feeds_analyzed_by_feed_press();
+ * ----------------------------------------------------------
+ * This function returns a list of feeds, which are analyzed by FeedPress AND are 
+ * provided by the present Podlove Publisher instance.
+ * 
+ * fetch_feed_press_feed_info( string $feedname );
+ * ----------------------------------------------------------
+ * This function returns advanced statistics for $feedname for the last 30 days.
+ * 
+ */
+
+namespace Podlove\Modules\FeedPress;
+use \Podlove\Model;
+use \Podlove\Http;
+
+class Feed_Press extends \Podlove\Modules\Base {
+
+    protected $module_name = 'FeedPress';
+    protected $module_description = 'FeedPress provides accurate and frequently updated analytics that bloggers and podcasters have come to trust. This module provides access the latest statistics.';
+    protected $module_group = 'external services';
+	
+    public function load() {
+        if ( isset( $_GET["page"] ) && $_GET["page"] == "podlove_settings_modules_handle") {
+            add_action('admin_bar_init', array( $this, 'check_code'));
+        }
+
+        $this->register_settings();
+    }
+
+    /**
+     * Fetches Statistics for a specified feed analyzed by FeedPress
+     *
+     * The Statistic Objects are returned with these properties:
+     *     - (int) day (Unix timestamp)
+     *     - (int) greader
+     *     - (int) other
+     *     - (int) direct
+     *     - (int) newsletter
+     * 
+     * @param  string $token    [FeedPress API Token)
+     * @param  string $feedname [Name of FeedPress Feed]
+     * 
+     * @return array            [Array of the Feed Stats for the last 30 days]
+     */
+    public static function fetch_feed_press_feed_info($feedname=FALSE) {
+        if ( ! $feedname || ! $token = get_option( 'podlove_module_feed_press' )['feed_press_api_key'] )
+            return;
+
+        $curl = new Http\Curl();
+        $curl->request( 'https://api.feed.press/feeds/subscribers.json?key=58eaa1be7018c&token=' . $token . '&feed=' . $feedname, array(
```

Comment:
This is not an API key. It is more something like a client ID as used in the Auphonic module 
https://github.com/podlove/podlove-publisher/blob/master/lib/modules/auphonic/auphonic.php#L44